bmi055: fixes for on-chip filter

bkueng opened the pull request:
While going through the BMI055 I noticed the on-chip filters are not configured as expected.
This changes:
- Accel: use cutoff of 62.5 Hz instead of 500 Hz
- Gyro: the cutoff frequency is coupled with the ODR and is fixed to 116 Hz
  at 1 kHz readout rate. So this PR does not change anything for the gyro.

Relevant parts from the datasheet:
![screenshot_20190211_150946](https://user-images.githubusercontent.com/281593/52568473-1ef0de80-2e0f-11e9-9987-cbdf6f7cc6be.png)

![screenshot_20190211_150929](https://user-images.githubusercontent.com/281593/52568464-1a2c2a80-2e0f-11e9-900c-4f8b243c3758.png)


@bresch can you test this on one of the vehicles?

dagar commented:
To be consistent with the other IMU drivers you should probably add the PX4 side low pass as well.

 - IMU_ACCEL_CUTOFF
 - IMU_GYRO_CUTOFF

I want to get this out of the actual drivers, but it's what we currently have.

dagar commented:
@bresch if you happen to still have a flight log from flying the BMI055 as primary with no filtering it might be interesting to compare. https://github.com/PX4/Firmware/pull/11432

davids5 commented on src/drivers/imu/bmi055/BMI055_accel.cpp:
```diff
@@ -214,48 +214,34 @@ BMI055_accel::probe()
-	/* Write accel ODR */
-	modify_reg(BMI055_ACC_BW, clearbits, setbits);
-
 	return OK;
```
@bkueng - with the removal of the setting of the register does this all become a NOP?

![image](https://user-images.githubusercontent.com/1945821/52590210-9c0f5a00-2df5-11e9-8c4e-94f31f738fa5.png)

bkueng commented:
> To be consistent with the other IMU drivers you should probably add the PX4 side low pass as well.

Right that is the other part that was missing. I added them as well.

> @bkueng - with the removal of the setting of the register does this all become a NOP?

Nice catch. I assumed it was used elsewhere. I removed it completely.

dagar commented:
Anyone able to give this a quick flight test on a pixhawk 4 with the icm20689 disabled?

dagar commented:
@PX4/testflights could you try this on a pixhawk 4 or 4 mini? Start with any normal flight as a baseline. Then repeat, but this time manually disable the primary IMU (try `mpu6000 stop`). If that doesn't work it can be done via parameter (CAL_ACC0_EN,  CAL_GYRO0_EN).

dannyfpv commented:
tested on pixhawk4 v5  normal flight
https://review.px4.io/plot_app?log=3a3ec430-965b-4421-ba1d-8204dd654dc5
disable the primary IMU stops and go on mission mode
https://review.px4.io/plot_app?log=44cb4cd5-fcc9-4e24-84d0-51587ac701bc

tested on pixhawk4 mini v5  normal flight
https://review.px4.io/plot_app?log=12d30737-48d5-41ab-b15a-5ec107b98dcc
disable the primary IMU stops and go on mission mode
https://review.px4.io/plot_app?log=042f3964-4836-4dd8-8ec1-ad4ca20c00c1

dagar commented:
The estimator really didn't like the data from the BMI055.

![image](https://user-images.githubusercontent.com/84712/52987836-d915b680-33ca-11e9-9031-08c10b90dfdd.png)

davids5 commented:
@dagar - what would you suggest?

dagar commented:
This is still an improvement over the existing state in master, but the bmi055 will need a closer look if we want it to truly be a safe fallback.
